redaxo/src/core/pages/setup.php

```diff
@@ -181,6 +181,14 @@
     $config['db'][1]['login'] = rex_post('redaxo_db_user_login', 'string');
     $config['db'][1]['password'] = rex_post('redaxo_db_user_pass', 'string');
     $config['db'][1]['name'] = rex_post('dbname', 'string');
+    $config['use_https'] = rex_post('use_https', 'string');
+    $config['use_hsts'] = rex_post('use_hsts', 'boolean');
+
+    if ($config['use_https'] === 'true') {
+        $config['use_https'] = true;
+    } elseif ($config['use_https'] === 'false') {
+        $config['use_https'] = false;
+    }
```

Comment:
Gibt es eine elegantere Variante damit `true` und `false` als Boolean, `frontend` und `backend` als String in die `config.yml` geschrieben werden?

Reply:
Hier sollte man noch checken dass nur einer der gültigen werte drinnen steht

Reply:
wird das nicht eigentlich schon durch das verwendete ´<select>´ mit den vorgegebenen Optionen gemacht?

Reply:
Das select und die option befinden sich auf dem client und bringt somit keine validierung. Never trust the client.

Ein user kann via browser dev-tools ein beliebiges eigenes select-option einfügen, mit beliebigem value